src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,155 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Note<T> first;
```

Comment:
Maybe you should rename this and other same fields/variables "NoDe" not "NoTe"  (at the moment you have both Note and Node on your code). Also, according to checklist we should make NoDe as inner class.